backend/app/models/inquiry.py

```diff
@@ -20,6 +26,11 @@ class Inquiry(InquiryBase, table=True):
     text: str = Field(min_length=10, max_length=255, unique=True)
     created_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
 
+    scheduled_inquiries: list["ScheduledInquiry"] = Relationship(
+        back_populates="inquiry"
+    )
+    responses: list["Response"] = Relationship(back_populates="inquiry")
```

Comment:
I just want to be clear on this.  
On Monday, the inquiry is given: "Do you love your boss"
On Thursday, the inquiry is given: "Do you love your boss"

Is the datetime on the Response what we are using to distinguish the Monday response from the Thursday response?

Reply:
This is not how we expected it to work. Let's discuss it on our 1:15 meeting today.